x/permission/exported/types_test.go

```diff
@@ -0,0 +1,41 @@
+package exported_test
+
+import (
+	"strings"
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/gogo/protobuf/protoc-gen-gogo/descriptor"
+	"github.com/stretchr/testify/assert"
+	"golang.org/x/exp/slices"
+
+	"github.com/axelarnetwork/axelar-core/app"
+	"github.com/axelarnetwork/axelar-core/x/permission/exported"
+)
+
+func TestMsgRoles(t *testing.T) {
+	registry := app.MakeEncodingConfig().InterfaceRegistry
+	implementations := registry.ListImplementations(sdk.MsgInterfaceProtoName)
+	slices.Sort(implementations)
+
+	var missingRoles []string
+	for _, implementation := range implementations {
+		if strings.HasPrefix(implementation, "/ibc") ||
+			strings.HasPrefix(implementation, "/cosmos") {
+			continue
+		}
+
+		msg, err := registry.Resolve(implementation)
+		assert.NoError(t, err)
+
+		role := exported.GetPermissionRole(msg.(descriptor.Message))
+		if role == exported.ROLE_UNSPECIFIED {
+			missingRoles = append(missingRoles, implementation)
+			continue
```

Comment:
there is no other instruction after this one regardless of the executed flow. Do we really need it here?

Reply:
ah there used to be more afterwards that was deleted. You are right, it's not needed anymore